Makefile

```diff
@@ -151,6 +151,11 @@ endif
 endif
 endif
 
+# TODO(codingllama): Build static binaries with libfido2.
+ifneq ("$(wildcard /usr/local/lib/libfido2.*)","")
```

Comment:
Perhaps we should make it more explicit that this is just for tests in buildbox? Hardcoding `/usr/local/lib` feels quite weird, otherwise.

Reply:
This works for buildbox and for local environments that have libfido2, Linux and macOS alike. I'll add a comment clarifying.

Reply:
`brew install libfido2`, which is what I would recommend (docs pending), links to /usr/local/lib too. I'm not sure about ARM, but we can tweak it accordingly.

> Are we going to require libfido2 to be installed externally for our macOS tsh?

Nope, I'll be static linking everything we need to `tsh` at some point.